### ISSUE TITLE
Changing sympa.example.com/wws/wsdl to sympa.example.com/wsdl gives 404

### ISSUE THREAD

**Issue by qulien** [sympa-community/sympa#1270]
I wanted to change the sympa setup so that instead of reaching the web interface at `http://sympa.example.com/wws`, I could reach it simply via `http://sympa.example.com`. I also simplified `http://sympa.example.com/sympasoap` to `http://sympa.example.com/soap`.

To do this, I changed the following lines in `/etc/sympa/sympa/sympa.conf`:

 - `wwsympa_url http://sympa.example.com/wws` → `wwsympa_url http://sympa.example.com/`
 - `soap_url http://sympa.example.com/sympasoap` → `soap_url http://sympa.example.com/soap`
 
And the following lines in `/etc/apache2/conf-available/sympa.conf` and `sympa-soap.conf` respectively:
 
 - `ScriptAlias /wws /usr/lib/cgi-bin/sympa/wwsympa-wrapper.fcgi` → `ScriptAlias / /usr/lib/cgi-bin/sympa/wwsympa-wrapper.fcgi`
 - `ScriptAlias /sympasoap /usr/lib/cgi-bin/sympa/sympa_soap_server-wrapper.fcgi` → `ScriptAlias /soap /usr/lib/cgi-bin/sympa/sympa_soap_server-wrapper.fcgi`
 
Everything seems to work as expected, with one exception:

 - `sympa.example.com/wsdl` gives a 404 Error instead of serving the WSDL definition.

Version
-------

 - Sympa 6.2.40

Installation method
-------------------

`apt install sympa`

Expected behavior
-----------------

 - WSDL description can be reached via `http://sympa.example.com`

Actual behavior
---------------
 
 - `http://sympa.example.com` yields 404 Error
 
Additional information
----------------------

I assume I'm missing something; would appreciate any pointers in the right direction.

**Comment by ikedas** [comment]
Hi @qulien ,

Did you setup Sympa SOAP service?  Please see [the documentation](https://sympa-community.github.io/manual/customize/soap-api.html#setup) for details.

**Comment by qulien** [comment]
@ikedas yep, SOAP is set up. API calls also work (when using the previous URL scheme, including `wws`). Just couldn't get it to work without the `wws` yet.

**Comment by racke** [comment]
What about a redirect / rewrite in your Apache configuration from wws/wsdl to wsdl?

**Comment by ikedas** [comment]
@qulien , please show the full configuration you made for Apache and Sympa.

**Comment by ikedas** [comment]
And just a thought, can you access to `http://sympa.example.com/lists` ?

**Comment by ikedas** [comment]
I decided this issue is a duplicate of #879.

mod_proxy_fcgi always determines _only one_ path component as SCRIPT_NAME CGI variable.  Thus, with @qulien 's setting, CGI variables are set as below:

  - `http://sympa.example.com/`:
    ```
    REQUEST_URI=/
    SCRIPT_NAME=/
    PATH_INFO=        (empty)
    ```

  - `http://sympa.example.com/wsdl`:
    ```
    REQUEST_URI=/wsdl
    SCRIPT_NAME=/wsdl
    PATH_INFO=        (empty)
    ```

  - `http://sympa.example.com/lists/others`:
    ```
    REQUEST_URI=/lists/others
    SCRIPT_NAME=/lists
    PATH_INFO=others
    ```
Though expected behavior is that `SCRIPT_NAME` will always be `/` and `PATH_INFO` will be remainder of the path.

This bug was fixed on #879 and it was included in Sympa 6.2.56.

So this issue is closed as a duplicate issue.
